SCClassLibrary/External/Ableton/SCClockMeterSync.sc

```diff
@@ -0,0 +1,246 @@
+// originally conceived for LinkClock (Ableton Link support)
+// but this may in theory apply to any clock that answers to
+// baseBarBeat_ (e.g. MIDISyncClocks, or BeaconClock)
+
+// TODO OSC message prefix with SC_
```

Comment:
this appears to be done now

Reply:
Ah, forgot to remove that. Will do in the next round.